interface/billing/freebtest.php

```diff
@@ -1,103 +0,0 @@
-<?php
-
```

Comment:
I still remember the joy I felt in the old days when I was actually able to get freeb/perl to work. Many, many, hours it took... @sunsetsystems